downloads.go

```diff
@@ -1,5 +1,9 @@
 package rtorrent
 
+import "net"
```

Comment:
These imports should be in a single block.

```
import (
    "net"
)
```

Tools like goimports do it for you.